vector/src/main/java/im/vector/riotx/core/utils/NoOpMatrixCallback.kt

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2020 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package im.vector.riotx.core.utils
+
+import im.vector.matrix.android.api.MatrixCallback
+
+class NoOpMatrixCallback<T>: MatrixCallback<T>
```

Comment:
Maybe move this class to the SDK (in file MatrixCallback.kt)?

Reply:
Yes it could

Reply:
done